src/components/ShareLinks/ShareLinks.tsx

```diff
@@ -0,0 +1,77 @@
+import './ShareLinks.scss';
+
+import { Flex, Icon, MarginSystemProps } from '@eleven-labs/design-system';
+import React from 'react';
+import {
+  FacebookIcon,
+  FacebookShareButton,
+  LinkedinIcon,
+  LinkedinShareButton,
+  RedditIcon,
+  RedditShareButton,
+  TwitterIcon,
+  TwitterShareButton,
+} from 'react-share';
+
+import { AVAILABLE_SHARE_LINKS } from '@/constants';
+import { useCopyToClipboard } from '@/hooks/useCopyToClipboard';
+
+export type ShareLinkOptions = {
+  urlToShare: string;
+};
+
+export type ShareLinksProps = MarginSystemProps & ShareLinkOptions;
+
+export const ShareLinks: React.FC<ShareLinksProps> = ({ urlToShare, ...flexProps }) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [_, copy] = useCopyToClipboard();
+  const getShareLinkIcon = (linkName: string, index: number): JSX.Element | undefined => {
+    switch (linkName) {
+      case 'copyLink':
```

Comment:
Todo in another PR => Add a notification in the Design System and display a notification to say that the link is copied
(Watch the behavior on medium)

Reply:
No need for that since I display a `copied` label when the copy url of the article button is clicked. Should suffice to indicate a well copied text